trl/trainer/ppo_trainer.py

```diff
@@ -1389,27 +1384,18 @@ def _show_tokens(self, tokens, masks):
                 text.append(" ")
         print(text)
 
-    def _prepare_deepspeed_zero3(self, model: PreTrainedModelWrapper):
+    def _prepare_deepspeed(self, model: PreTrainedModelWrapper):
         # Adapted from accelerate: https://github.com/huggingface/accelerate/blob/739b135f8367becb67ffaada12fe76e3aa60fefd/src/accelerate/accelerator.py#L1473
-        # TODO: figure out if any other parameters are needed to optimize inference
         deepspeed_plugin = self.accelerator.state.deepspeed_plugin
-        batch_size_per_device = deepspeed_plugin.deepspeed_config["train_micro_batch_size_per_gpu"]
-        # See DeepSpeed docs for definition of these parameters: https://deepspeed.readthedocs.io/en/latest/zero3.html
-        config_kwargs = {
```

Comment:
All these parameters are set automatically by `accelerate` and this don't need duplicating. One check I need to make is the inclusion of gradient accumulation